### PR TITLE
Vs2019 support and dropping VS2015 support

### PR DESCRIPTION
**Pull request by UnityAlex**

### CODE REVIEW

**Comment by UnityAlex** [comment]
@joncham no rush just doing some housekeeping before I forget. Wanted to get this merged and the README updated to reflect the current state of the plugin.

File: MigrationBackup/f60f1f98/UnityMixedCallStack/NuGetUpgradeLog.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
```
Comment (joncham) [inline MigrationBackup/f60f1f98/UnityMixedCallStack/NuGetUpgradeLog.html]: I don't think we need this.

Comment (UnityAlex) [reply MigrationBackup/f60f1f98/UnityMixedCallStack/NuGetUpgradeLog.html]: ah yeah true. I'll remove it.

**Review by joncham — APPROVED** [review]
Looks good. Maybe squash commits into one before or as part of merge?

**Pull request merged**